### PR TITLE
ERC1155 examples

### PR DESCRIPTION
**Pull request by KarishmaBothara**
## Summary

Closes https://centralitydev.atlassian.net/browse/TRN-197

## Checklist

- [x] Add description
- [x] Tag related issue(s)

### CODE REVIEW

File: examples/evm/use-erc1155/README.md
Comment (ken-centrality) [inline examples/evm/use-erc1155/README.md:1]: Remove the extrinsic part of the create collection as this already cover by `substrate/use-sft` example I believe

File: packages/utils/src/getERC1155PrecompileContract.ts [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+export const getERC1155Precompile = (
+  privateKey: string,
+  precompileAddress: string | null,
+  collectionId: string | number | null
+) => {
+  // const provider = getLocalProvider(false).provider;
+  const wallet = new Wallet(
+    privateKey,
+    getDefaultProvider(`http://127.0.0.1:9933`)
```
Comment (ken-centrality) [inline packages/utils/src/getERC1155PrecompileContract.ts:69]: use `getEthersProvider` in the `utils` package

File: packages/utils/src/getSFTPrecompile.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+export const SFT_PRECOMPILE_ABI = [
+  "event InitializeSftCollection(address indexed collectionOwner, address indexed precompileAddress)",
+  "function initializeCollection(address owner, bytes name, bytes metadataPath, address[] royaltyAddresses, uint32[] royaltyEntitlements) returns (address, uint32)",
+];
+
+export const getSFTPrecompile = (privateKey: string) => {
+  // const provider = new JsonRpcProvider(`http://127.0.0.1:9933`);
+  const wallet = new Wallet(
+    privateKey,
+    getDefaultProvider(`http://127.0.0.1:9933`)
```
Comment (ken-centrality) [inline packages/utils/src/getSFTPrecompile.ts:17]: ditto, use `getEthersProvider`

File: examples/evm/use-erc1155/README.md
Comment (KarishmaBothara) [reply examples/evm/use-erc1155/README.md:1]: ok

File: packages/utils/src/getERC1155PrecompileContract.ts (hunk 1 above)
Comment (KarishmaBothara) [reply packages/utils/src/getERC1155PrecompileContract.ts:69]: this one is not on porcini yet.. and the func getEthersProvider accept network name

```
export function getEthersProvider(network: NetworkName) {
  return getDefaultProvider(getPublicProviderUrl(network));
}
```
Once this commit is deployed on porcini.. I can update the code to use porcini

**Review by ken-centrality — APPROVED** [review]

**Pull request merged**